src/gl/contourFeature.js

```diff
@@ -103,7 +103,9 @@ var gl_contourFeature = function (arg) {
           '    } else {',
           '      step = valueVar;',
           '    }',
-          '    clr = texture2D(sampler2d, vec2(step / steps, 0.0));',
+          // our texture is padded on either end by a repeated value to ensure
+          // we interpolate smoothly at the ends.
+          '    clr = texture2D(sampler2d, vec2((step + 1.0) / (steps + 2.0), 0.0));',
```

Comment:
can we not use the GL texture GL_CLAMP_TO_EDGE which would probably give us the same behavior?

Reply:
not sure, I will have a look but for now this is good.